index.html

```diff
@@ -99,9 +99,9 @@
             </div>
 
             <div class="links">
-                <a href="https://klink.asia">News</a>
-                <a href="https://docs.klink.asia">Documentation</a>
-                <a href="https://git.klink.asia/main/k-link">Contribute</a>
+                <a href="https://k-link.technology">News</a>
+                <a href="https://k-link.technology">Documentation</a>
```

Comment:
Why do we take the link to the docs away?

Reply:
Contains outdated materials and do not point to the correct domain. I would strip it out, or make it pointing to GitHub for now